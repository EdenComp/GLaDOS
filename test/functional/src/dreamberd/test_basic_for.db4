for(int i = 0; i < 3; i++) {
    print("This for works!");
}