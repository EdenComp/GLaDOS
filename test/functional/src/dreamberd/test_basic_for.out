This for works!This for works!This for works!